.travis.yml

```diff
@@ -7,6 +7,12 @@ branches:
 
 before_install:
   - pip install --upgrade pip setuptools wheel
+  - sudo apt-get install openjdk-8-jdk
```

Comment:
It's actually this difficult to install bazel?

Reply:
Yes. It's kind of ridiculous. https://docs.bazel.build/versions/master/install-ubuntu.html. There might be a `java` already installed in Travis so the first line is probably redundant, but we still need to add the distribution.

Reply:
Everything worked out. Afraid to rock the boat since the Bazel distro we are using is specific to `jdk1.8`.